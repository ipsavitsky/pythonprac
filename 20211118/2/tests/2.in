class C:
    n = Num()

c = C()

print(c.n)
c.n = range(1000)
print(c.n)
del c.n
print(c.n)