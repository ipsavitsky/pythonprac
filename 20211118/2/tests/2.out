0
1000
None
